dashboard/app/controllers/levels_controller.rb

```diff
@@ -428,6 +428,7 @@ def level_params
       {if_block_options: []},
       {place_block_options: []},
       {play_sound_options: []},
+      {visible: {}},
```

Comment:
maybe put a comment here that this is CSA-specific? right now it looks like it's in the Minecraft-specific category

Reply:
Additionally, the variable name makes it appear that we're defining whether the level itself is visible. Suggested renames: `start_sources_metadata`, `files_metadata`, `visible_files`, `file_view_types` or something similar.